src/db.js

```diff
@@ -12,9 +12,9 @@ class Db {
     firebase.initializeApp(config.firebase);
     firebase.auth().signInAnonymously();
     this.db = firebase.firestore();
-    this.db.settings({
-      timestampsInSnapshots: true,
-    });
+    // this.db.settings({
```

Comment:
Is this settning not needed for firestore anymore?

Reply:
Exactly, this is a default in the latest update. I'm removing the comment.